pkg/audit/manager.go

```diff
@@ -883,7 +886,7 @@ func (am *Manager) addAuditResponsesToUpdateLists(
 		if len(msg) > msgSize {
 			msg = truncateString(msg, msgSize)
 		}
-		action := string(ea)
+		action := strings.Join(r.EnforcementAction, "/")
```

Comment:
I think Rita mentioned that changing this to a "/" separated string breaks backwards compatibility?